dataclients/kubernetes/definitions/ingressvalidator.go

```diff
@@ -0,0 +1,68 @@
+package definitions
+
+import (
+	"fmt"
+
+	"github.com/zalando/skipper/eskip"
+)
+
+const (
+	skipperfilterAnnotationKey    = "zalando.org/skipper-filter"
+	skipperpredicateAnnotationKey = "zalando.org/skipper-predicate"
+	skipperRoutesAnnotationKey    = "zalando.org/skipper-routes"
+)
+
+type IngressV1Validator struct{}
+
+var defaultIngressV1Validator = IngressV1Validator{}
```

Comment:
hmm why do we use a package global here?

Reply:
Because later we will need to use this in webhook & ingress definitions and maybe custmoize with filter registry and so on